src/Common/src/CoreLib/System.Private.CoreLib.Shared.projitems

```diff
@@ -731,6 +731,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Numerics\Vector_Operations.cs" />
   </ItemGroup>
   <ItemGroup Condition="$(TargetsWindows)">
+    <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Advapi32\Interop.RegistryValues.cs" />
```

Comment:
Can this be in the block with other registry APIs?

Reply:
This file was earlier included irrespective of the value of enableWinRt variable where as all other registry files are include is this property is not true ?
is it fine to move there ?

Reply:
It should be fine. I do not see any problem with it.